### PR TITLE
UI/fixing showpage mobilescreen size and updated map width accordingly.

### PR DESCRIPTION
**Pull request by preranah7**
📋 Description
This Pull Request ensures that the user can view the images clearly and in bigger height and width on moblie screen size.
Earlier the images their description and map was viewed column-wise which was not completing the usage of images completely.

🔨 Changes Made
Made the components view row-wise
Fixed the map width for mobile screen size through CSS media-queries
✅ Checklist
Before submitting the PR, please make sure you have completed the following:

[✅] My code follows the style guidelines of this project.
[✅] I have performed a self-review of my own code.
[✅] I have commented my code, particularly in hard-to-understand areas.
 I have made corresponding changes to the documentation (if applicable).
[✅] My changes generate no new warnings.
[✅] I have tested it locally and it works fine.
 Any dependent changes have been merged and published in downstream modules.
🏷️ Types of Changes
What type of changes does your code introduce? (Check all that apply)

 Bug fix (non-breaking change which fixes an issue) 🐛
 New feature (non-breaking change which adds functionality) ✨
[✅] UI enhancement (non-breaking change which enhances UI) 🎨
 Documentation update 📚

🤝 Related Issues
Fixes [UI ENHANCEMENT] make changes in the show page for mobile screen size. #65

Kindly review my changes.
Please tell me if it produces any errors.
Thankyou.

### CODE REVIEW

File: public/stylesheets/app.css [hunk 1]
```diff
@@ -58,4 +58,27 @@ h1{
...
+
+    .col-6 {
+        width: 100%;
+        height: auto;
+    }
+
+    .row {
+        flex-direction: column !important;
+    }
+}
```
Comment (Vignesh025) [inline public/stylesheets/app.css:84]: This is not necessary.

File: views/campgrounds/show.ejs [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <%layout('layouts/boilerplate')%>
     <link rel="stylesheet" href="/stylesheets/stars.css">
 
-    <div class="row" id="showinfo">
-        <div class="col-6">
+    <div class="row flex-column flex-md-row" id="showinfo">
+        <div class="col-12 col-md-6 mb-3">
```
Comment (Vignesh025) [inline views/campgrounds/show.ejs:5]: These many classes are not necessary.

File: views/campgrounds/show.ejs [hunk 3]
```diff
@@ -1,8 +1,8 @@
 <%layout('layouts/boilerplate')%>
     <link rel="stylesheet" href="/stylesheets/stars.css">
 
-    <div class="row" id="showinfo">
-        <div class="col-6">
```
Comment (Vignesh025) [inline views/campgrounds/show.ejs:5]: Replace col-6 with col-md-6 in line number 5 and 56.
This will fix the issue.

**Review by Vignesh025 — COMMENTED** [review]
Make the necessary changes and open the PR

File: views/campgrounds/show.ejs (hunk 3 above)
Comment (preranah7) [reply views/campgrounds/show.ejs:5]: Sure , I will do it again.
I am sorry for the unnecessary addition of lines of code I thought was necessary.
Thankyou for your detailed explanation.

Comment (Vignesh025) [reply views/campgrounds/show.ejs:5]: No worries at all, we all learn together from our mistakes. Keep going!

Comment (preranah7) [reply views/campgrounds/show.ejs]: I have edited the files. 
Please let me know is this okay or should I create a new PR for the changes you explained.

Comment (Vignesh025) [reply views/campgrounds/show.ejs]: There is no need to create a new PR, this is good!

**Comment by Vignesh025** [comment]
*Thankyou for Contributing!*

**Pull request merged**